ui/templates/importer/upload.html

```diff
@@ -0,0 +1,9 @@
+{% extends "lore/base.html" %}
```

Comment:
I removed `lore/` and it broke it. I think it's because they're under different subfolders in `ui/templates`.